static/styles/settings.css

```diff
@@ -365,7 +365,7 @@ td .button {
 }
 
 .dependent-block {
-    margin: -5px 0 15px 23px;
+    margin: 15px 0 -5px 23px !important;
```

Comment:
Why do we need `!important` here? I wonder if it'd be better to adjust the element's selector approach instead.